src/lib.rs

```diff
@@ -16,6 +16,20 @@
 //!
 //! ```no_run
 //! let mut roots = rustls::RootCertStore::empty();
+//! let root_certs = &rustls_native_certs::load_native_certs()
+//!     .expect("could not load platform certs")
+//!     .iter()
+//!     .map(|c| c.0.clone())
+//!     .collect::<Vec<Vec<u8>>>();
+//!
+//! roots.add_parsable_certificates(root_certs);
```

Comment:
There might be an argument to be made that we should do something with the return from `add_parseable_certificates` to make it more obvious that this function returns useful information about how many roots were added vs ignored.

I've left it out to keep the example succinct. WDYT?